src/Hedgehog/Internal/Function.idr

```diff
@@ -0,0 +1,351 @@
+module Hedgehog.Internal.Function
+
+import Data.Colist
+import Data.Cotree
+import Data.Either
+import Data.String
+
+import Hedgehog.Internal.Gen
+import Hedgehog.Internal.Range
+import Hedgehog.Internal.Seed
+
+import Language.Reflection.Util
```

Comment:
Done. However, I don't understand why the convention is to reexport some stuff that is essential to a `derive` function work by a `Derive.*` module rather than by `Language.Reflection.Derive` (like, say, reexporting `Data.List1` and `Data.Vect`). I consider this to be a non-obvious implementation detail of the `derive` function, and making every `Derive.*` module to reexport the detail of implementation of `Language.Reflection.Derive` module seems strange to me.